src/Slider/Slider.tsx

```diff
@@ -35,11 +35,17 @@ export interface SliderProps {
 const Slider: React.FunctionComponent<SliderProps> = (props: SliderProps): React.ReactElement<void> => {
     const [min, setMin] = React.useState<number>(props.min || 0);
     const [max, setMax] = React.useState<number>(props.max || 100);
+    const [size, setSize] = React.useState<number>(0);
     const [labelsPositions, setLabelsPositions] = React.useState<Array<string>>([]);
+    const [thumbPosition, setThumbPosition] = React.useState<number>(0);
 
     React.useEffect(() => {
-        setMin(props.min || 0);
-        setMax(props.max || 100);
+        // Checking if the min or max are not numbers, null value or undefined
+        const minValue: number = typeof props.min !== "number" ? 0 : props.min;
```

Comment:
typeof NaN is also a number, maybe we should add IsNaN
```suggestion
        const minValue: number = (typeof props.min !== "number" && isNaN(props.min)) ? 0 : props.min;
```

Reply:
`typeof variable` is a more accurate than `isNaN` to what I'm trying to achieve given the following:

| value | `isNaN` | `typeof test !== "number"` |
| -- | -- | -- |
| `null` | `false` | `true` |
| `"1"` | `false` | `true` |
| `"+1"` | `false` | `true` |
| `""` | `false` | `true` |
| `new Date()` | `false` | `true` |
| `[]` | `false` | `true` |